include/openthread/platform/radio.h

```diff
@@ -276,10 +279,15 @@ typedef struct otRadioFrame
              * Note: `mTxDelayBaseTime` + `mTxDelay` SHALL point to the point in
              * time when the end of the SFD will be present at the local
              * antenna, relative to the local radio clock.
+             *
+             * This field together with mTxDelayBaseTime should be orthogonal to mCsmaCaEnabled. If this field is
+             * non-zero, mMaxCsmaBackoffs should be ignored.
              */
             uint32_t mTxDelay;
 
-            uint8_t mMaxCsmaBackoffs; ///< Maximum number of backoffs attempts before declaring CCA failure.
+            uint8_t mMaxCsmaBackoffs; ///< Maximum number of backoffs attempts before declaring CCA failure, 0 to
+                                      ///< transmit the frame without CSMA backoff. This field should be ignored if
+                                      ///< mCsmaCaEnabled is false or either mTxDelayBaseTime or mTxDelay is non-zero.
```

Comment:
Same here? 
```c++
            /**
             * Maximum number of CSMA backoff attempts before declaring channel access failure.
             *
             * This is applicable and MUST be used when radio platform provides the `OT_RADIO_CAPS_CSMA_BACKOFF` 
             * capability. 
             * 
             * This field MUST be ignore if `mCsmaCaEnabled` is set to `false` (i.e. CCA is disabled) or either 
             * `mTxDelayBaseTime` or `mTxDelay` is non-zero (frame transmission is expected at a specific time).
             *
             * It can be set to `0` to skip backoff mechanism (note that CCA MUST still be performed assuming 
             * `mCsmaCaEnabled` is `true`).
             */ 
            uint8_t mMaxCsmaBackoffs; 
```